### PR TITLE
Adds func for reading multi and flat genbank files

### LINKED ISSUE #75: NCBI Genbank flat file parsing
**Is your feature request related to a problem? Please describe.**
Downloading bulk Genbank from https://ftp.ncbi.nlm.nih.gov/genbank/ gives a little bit of a unique file format. There are many sequences within each file, and they start with a header. I would like to be able to directly pipe those into Poly for manipulation. Specifically, I am looking at building codontable.com, and this kind of data parsing will be necessary. 

**Describe the solution you'd like**
1. Proper parsing of multi-genbank files (files with more than 1 genbank file)
2. Parsing of the Genbank flat file header

**Describe alternatives you've considered**
I've considered doing a real dirty bash parsing to get the job done, but it's probably worthwhile to make the Genbank flat file parser. It *will* tease out any bugs in the original iteration of the Genbank parser since it can be used on all official Genbank files.

**Additional context**
Genbank files are separated by //

The headers of Genbank flat files look like:
```
GBBCT1.SEQ          Genetic Sequence Data Bank
                         October 15 2020

                NCBI-GenBank Flat File Release 240.0

                     Bacterial Sequences (Part 1)

  101593 loci,   185853961 bases, from   101593 reported sequences


```
with LOCUS immediately following.

### PR DESCRIPTION
**Pull request by Koeng101**
Closes #75 

Adds functionality to parse and read multi-genbank files and genbank flat files (like the ones exported from the Genbank FTP server). 

I did NOT add Build or Write functions for these 2 - I wasn't quite sure how they'd fit into the rest of the Poly ecosystem, but these seemed easy enough to add, and give functionality necessary for testing out how Poly operates when given ALL OF GENBANK.

### CODE REVIEW

File: io.go [hunk 1]
```diff
@@ -1402,3 +1403,66 @@ func generateWhiteSpace(length int) string {
 GBK specific IO related things end here.
 
 ******************************************************************************/
+
+/******************************************************************************
+
+Genbank Flat specific IO related things begin here.
+
+******************************************************************************/
+
```
Comment (TimothyStiles) [inline io.go:1412]: Needs comment that will render in docs.

File: io.go [hunk 2]
```diff
@@ -1402,3 +1403,66 @@ func generateWhiteSpace(length int) string {
...
+	gbkWithoutHeader := []byte(strings.Join(strings.Split(gbk, "\n")[10:], "\n"))
+
+	// Pass gbkWithoutHeader to ParseGbkMulti, which should handle
+	// the rest of the parsing just fine
+	sequences := ParseGbkMulti(gbkWithoutHeader)
+	return sequences
+}
+
+// Parse genbank files with multiple genbank files
+func ReadGbkMulti(path string) []Sequence {
```
Comment (TimothyStiles) [inline io.go:1451]: Every function comment must start with `Function Name` followed by a comment.

File: io.go [hunk 3]
```diff
@@ -1402,3 +1403,66 @@ func generateWhiteSpace(length int) string {
...
+	file, err := ioutil.ReadFile(path)
+	if err != nil {
+		// return 0, fmt.Errorf("Failed to open file %s for unpack: %s", gzFilePath, err)
+	}
+	sequences := ParseGbkMulti(file)
+	return sequences
+}
+
+// Parse GbkFlat files
```
Comment (TimothyStiles) [inline io.go:1460]: Every function comment must start with `Function Name` followed by a comment. Also this reads not parses.

Comment (Koeng101) [reply io.go]: Oh yea, it does read, does not parse. Fixed!

**Review by TimothyStiles — APPROVED** [review]

**Pull request merged**